app/helpers/application_helper.rb

```diff
@@ -390,7 +389,19 @@ def server_plugin_outlet(name)
     return "" if erbs.blank?
 
     result = +""
-    erbs.each { |erb| result << render(inline: File.read(erb)) }
+    unless var
```

Comment:
not sure if we are going to merge this, but we certainly should never ever do `unless ... else` ... `if ... else` is far less confusing.